### ISSUE TITLE
Problem with dotnet version dependency

### ISSUE THREAD

**Issue by SignalMQ** [AvaloniaUI/AvaloniaVSCode#77]
### Describe the bug

I encountered this problem before launching the previewer. I initially installed dotnet-sdk-8.0 from the terminal. The extension in the logs required version sdk-7.0, after seeing this log I installed 7.0. Then the extension detected this SDK and started the build process, but then another error appeared in the logs, the target platform of the project is 6.0, and with that I decided to submit this bug with dependencies.

### To Reproduce

1. Create Avalonia MVVM project
2. Open directory of this project
3. Open MainWindow.xaml
4. Tap to design previewer button
5. Avalonia ask to build solution project
6. Tap to Build button
7. See logs below

### Avalonia for VS Code

v0.0.25

### Avalonia version

v11.0.5

### VS Code version

1.84.2

### Relevant log output

```shell
2023-12-01 09:13:54.491 [info] Creating language service
2023-12-01 09:13:54.491 [info] Starting Avalonia Language Server...
2023-12-01 09:13:54.491 [info] [Trace - 9:13:54 AM] Sending request 'initialize - (0)'.
2023-12-01 09:13:56.406 [info] [Trace - 9:13:56 AM] Received response 'initialize - (0)' in 2046ms.
2023-12-01 09:13:56.407 [info] [Trace - 9:13:56 AM] Sending notification 'initialized'.
2023-12-01 09:13:56.410 [info] [Trace - 9:13:56 AM] Sending notification 'workspace/didChangeConfiguration'.
2023-12-01 09:13:56.529 [info] [Trace - 9:13:56 AM] Received request 'client/registerCapability - (1)'.
2023-12-01 09:13:56.529 [info] [Trace - 9:13:56 AM] Sending response 'client/registerCapability - (1)'. Processing request took 2ms
2023-12-01 09:13:56.535 [info] [Trace - 9:13:56 AM] Received request 'workspace/configuration - (2)'.
2023-12-01 09:13:56.535 [info] [Trace - 9:13:56 AM] Sending response 'workspace/configuration - (2)'. Processing request took 1ms
2023-12-01 09:13:56.599 [info] [Trace - 9:13:56 AM] Received request 'workspace/configuration - (3)'.
2023-12-01 09:13:56.599 [info] [Trace - 9:13:56 AM] Sending response 'workspace/configuration - (3)'. Processing request took 0ms
2023-12-01 09:14:29.566 [info] [Trace - 9:14:29 AM] Sending notification 'textDocument/didOpen'.
2023-12-01 09:14:29.622 [info] [Trace - 9:14:29 AM] Received request 'workspace/configuration - (4)'.
2023-12-01 09:14:29.622 [info] [Trace - 9:14:29 AM] Sending response 'workspace/configuration - (4)'. Processing request took 1ms
2023-12-01 09:14:29.679 [info] [Trace - 9:14:29 AM] Received notification 'window/logMessage'.
2023-12-01 09:14:29.679 [info] [Info  - 9:14:29 AM] AvaloniaLanguageServer.Handlers.TextDocumentSyncHandler: ** DidOpenText: file:///home/sobirjon/Documents/Example/Views/MainWindow.axaml | Uri='file:///home/sobirjon/Documents/Example/Views/MainWindow.axaml'
2023-12-01 09:14:33.255 [info] Command avalonia.previewProcess, file:///home/sobirjon/Documents/Example/Views/MainWindow.axaml
2023-12-01 09:14:35.670 [info] [EXT - INFO] Solution data path path: /tmp/Example.sln.json
2023-12-01 09:14:35.719 [info] parser process args: /bin/sh,-c,dotnet "/home/sobirjon/.vscode/extensions/avaloniateam.vscode-avalonia-0.0.25/solutionParserTool/SolutionParser.dll" "/home/sobirjon/Documents/Example/Example.sln" --sdk 7.0
2023-12-01 09:14:35.722 [info] You must install or update .NET to run this application.

App: /home/sobirjon/.vscode/extensions/avaloniateam.vscode-avalonia-0.0.25/solutionParserTool/SolutionParser.dll
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '7.0.0' (x64)
.NET location: /usr/lib/dotnet/

The following frameworks were found:
  8.0.0 at [/usr/lib/dotnet/shared/Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=7.0.0&arch=x64&rid=ubuntu.23.10-x64&os=ubuntu.23.10

2023-12-01 09:14:35.727 [info] parser process exited 150
2023-12-01 09:17:15.173 [info] Command avalonia.previewProcess, file:///home/sobirjon/Documents/Example/Views/MainWindow.axaml
2023-12-01 09:17:17.776 [info] [EXT - INFO] Solution data path path: /tmp/Example.sln.json
2023-12-01 09:17:17.829 [info] parser process args: /bin/sh,-c,dotnet "/home/sobirjon/.vscode/extensions/avaloniateam.vscode-avalonia-0.0.25/solutionParserTool/SolutionParser.dll" "/home/sobirjon/Documents/Example/Example.sln" --sdk 7.0
2023-12-01 09:17:18.222 [info] parser process exited 255
2023-12-01 09:17:54.807 [info] Command avalonia.previewProcess, file:///home/sobirjon/Documents/Example/Views/MainWindow.axaml
2023-12-01 09:18:03.541 [info] [EXT - INFO] Solution data path path: /tmp/Example.sln.json
2023-12-01 09:18:03.612 [info] parser process args: /bin/sh,-c,dotnet "/home/sobirjon/.vscode/extensions/avaloniateam.vscode-avalonia-0.0.25/solutionParserTool/SolutionParser.dll" "/home/sobirjon/Documents/Example/Example.sln" --sdk 7.0
2023-12-01 09:18:03.993 [info] parser process exited 255
2023-12-01 09:19:21.888 [info] Command avalonia.previewProcess, file:///home/sobirjon/Documents/Example/Views/MainWindow.axaml
2023-12-01 09:19:24.206 [info] [EXT - INFO] Solution data path path: /tmp/Example.sln.json
2023-12-01 09:19:24.262 [info] parser process args: /bin/sh,-c,dotnet "/home/sobirjon/.vscode/extensions/avaloniateam.vscode-avalonia-0.0.25/solutionParserTool/SolutionParser.dll" "/home/sobirjon/Documents/Example/Example.sln" --sdk 7.0
2023-12-01 09:19:25.623 [info] parser process exited 0
2023-12-01 09:19:26.801 [info]   Determining projects to restore...

2023-12-01 09:19:27.484 [info]   All projects are up-to-date for restore.

2023-12-01 09:19:32.705 [info]   Example -> /home/sobirjon/Documents/Example/bin/Debug/net6.0/Example.dll

2023-12-01 09:19:32.732 [info] 

2023-12-01 09:19:32.733 [info] Build succeeded.

2023-12-01 09:19:32.734 [info]     0 Warning(s)
    0 Error(s)

2023-12-01 09:19:32.734 [info] 
Time Elapsed 00:00:06.71

2023-12-01 09:19:32.789 [info] Previewer assets generated at /home/sobirjon/.nuget/packages/avalonia/11.0.5/tools/netcoreapp2.0/designer/Avalonia.Designer.HostApp.dll
2023-12-01 09:19:32.791 [info] Command avalonia.previewProcess, file:///home/sobirjon/Documents/Example/Views/MainWindow.axaml
2023-12-01 09:19:32.793 [info] PreviewServer.start /home/sobirjon/Documents/Example/bin/Debug/net6.0/Example.dll
2023-12-01 09:19:32.808 [info] Previewer process started with args: exec,--runtimeconfig "/home/sobirjon/Documents/Example/bin/Debug/net6.0/Example.runtimeconfig.json",--depsfile "/home/sobirjon/Documents/Example/bin/Debug/net6.0/Example.deps.json" "/home/sobirjon/.nuget/packages/avalonia/11.0.5/tools/netcoreapp2.0/designer/Avalonia.Designer.HostApp.dll",--method avalonia-remote,--transport tcp-bson://127.0.0.1:8001/,--method html,--html-url http://127.0.0.1:8000,"/home/sobirjon/Documents/Example/bin/Debug/net6.0/Example.dll"
2023-12-01 09:19:32.809 [info] Preview server listening on port 8001
2023-12-01 09:19:32.811 [info] Show Preview to side: file:///home/sobirjon/Documents/Example/Views/MainWindow.axaml
2023-12-01 09:19:32.813 [info] You must install or update .NET to run this application.

App: /home/sobirjon/.nuget/packages/avalonia/11.0.5/tools/netcoreapp2.0/designer/Avalonia.Designer.HostApp.dll
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '6.0.0' (x64)
.NET location: /usr/lib/dotnet/

The following frameworks were found:
  7.0.14 at [/usr/lib/dotnet/shared/Microsoft.NETCore.App]
  8.0.0 at [/usr/lib/dotnet/shared/Microsoft.NETCore.App]

Learn more:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=6.0.0&arch=x64&rid=ubuntu.23.10-x64&os=ubuntu.23.10

2023-12-01 09:19:32.814 [info] Previewer process exited with code 150
```

### Additional context

![image](https://github.com/AvaloniaUI/AvaloniaVSCode/assets/77734001/79f1c1ee-7c2d-447c-99c4-a6636c122a98)

**Comment by tomschrot** [comment]
Yeah
The templates are a bit outdated.
Open the corresponding project file (.csproj).
Check line #4

`
<TargetFramework>net6.0</TargetFramework>
`

set the desired framework, e.g. 

`
<TargetFramework>net8.0</TargetFramework>
`

While you're at it, check & update the Avalonia rferences, e.g.

```
<ItemGroup> 
      <PackageReference Include="Avalonia" Version="11.0.5" />
      <PackageReference Include="Avalonia.Controls.DataGrid" Version="11.0.5" />
      <PackageReference Include="Avalonia.Desktop" Version="11.0.5" /> 
      <PackageReference Include="Avalonia.Themes.Fluent" Version="11.0.5" /> 
      <PackageReference Include="Avalonia.Fonts.Inter" Version="11.0.5" /> 
      <PackageReference Include="Material.Avalonia.DataGrid" Version="3.1.0" /> 
      <!--Condition below is needed to remove Avalonia.Diagnostics package from build output in Release configuration.--> 
      <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="11.0.5" /> 
</ItemGroup>
```
;)

**Comment by SignalMQ** [comment]
> Yeah The templates are a bit outdated. Open the corresponding project file (.csproj). Check line #4
> 
> `<TargetFramework>net6.0</TargetFramework>`
> 
> set the desired framework, e.g.
> 
> `<TargetFramework>net8.0</TargetFramework>`
> 
> While you're at it, check & update the Avalonia rferences, e.g.
> 
> ```
> <ItemGroup> 
>       <PackageReference Include="Avalonia" Version="11.0.5" />
>       <PackageReference Include="Avalonia.Controls.DataGrid" Version="11.0.5" />
>       <PackageReference Include="Avalonia.Desktop" Version="11.0.5" /> 
>       <PackageReference Include="Avalonia.Themes.Fluent" Version="11.0.5" /> 
>       <PackageReference Include="Avalonia.Fonts.Inter" Version="11.0.5" /> 
>       <PackageReference Include="Material.Avalonia.DataGrid" Version="3.1.0" /> 
>        
>       <PackageReference Condition="'$(Configuration)' == 'Debug'" Include="Avalonia.Diagnostics" Version="11.0.5" /> 
> </ItemGroup>
> ```
> 
> ;)

All works after I changed to .NET 7.0, but not works in versions 6.0, 8.0. In extension description is mentioned to require .NET 7.0.

**Comment by prashantvc** [comment]
Fixed in v0.0.26
